ready_recipes/arm_pyart/meta.yaml

```diff
@@ -0,0 +1,40 @@
+{% set version = "1.6.0" %}
+
+package:
+    name: arm_pyart
+    version: {{ version }}
+
+source:
+    fn: arm_pyart-{{ version }}.tar.gz
+    url: https://pypi.python.org/packages/source/a/arm_pyart/arm_pyart-{{ version }}.tar.gz
+    md5: 6c382dda78193c730a6a47d2d7b18eeb
+
+build:
+    number: 0
+
+requirements:
+    build:
+        - python
+        - numpy x.x
+        - trmm_rsl   # [not win]
+
+    run:
+        - python
+        - numpy
```

Comment:
> conda should still enforce the version requirement that is defined in the package name.

I don't think it does. The build string and the requirements are treated separately, and all the build string does is give you unique names AFIK.